ports/termcolor/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "termcolor",
+  "version-string": "1.0.1",
+  "homepage": "https://github.com/ikalnytskyi/termcolor",
+  "description": "Termcolor is a header-only C++ library for printing colored messages to the terminal."
```

```suggestion
  "description": "Termcolor is a header-only C++ library for printing colored messages to the terminal.",
  "homepage": "https://github.com/ikalnytskyi/termcolor"
```
Sorry, the order should be like this.

---

Thanks, I have verified that the order is right now.